### PR TITLE
fix(helm-sys): Double the helm timeout after reported timeout errors

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
# Description

There have been multiple reports of errors when deploying to some clusters.

image

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

### CODE REVIEW

File: rust/helm-sys/go-helm-wrapper/main.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ func main() {
 func go_install_helm_release(releaseName *C.char, chartName *C.char, chartVersion *C.char, valuesYaml *C.char, namespace *C.char, suppressOutput bool) *C.char {
 	helmClient := getHelmClient(namespace, suppressOutput)
 
-	timeout, _ := time.ParseDuration("10m")
+	timeout, _ := time.ParseDuration("20m")
```
Comment (razvan) [inline rust/helm-sys/go-helm-wrapper/main.go:42]: I'm not a go expert but I think this is the wrong timeout to change. Judging by the error message, I believe the context passed to `InstallChart() ` is the proper one to update.

Comment (Techassi) [reply rust/helm-sys/go-helm-wrapper/main.go:42]: The timeout passed to the `ChartSpec`

> configures the time to wait for any individual Kubernetes operation (like Jobs for hooks).

_(per documentation, see [here](https://pkg.go.dev/github.com/mittwald/go-helm-client#ChartSpec))_

The `context.Background()` function returns a context, which will never cancel. We could pass a timeout here as well, but that would configure the timeout for the _complete_ operation, not individual K8s operations.

So I think the change looks fine in my opinion.

**Review by razvan — APPROVED** [review]
I guess it's worth a try.

**Comment by NickLarsenNZ** [comment]
> I guess it's worth a try.

That's the spirit :smile:

**Pull request merged**